include/openmc/constants.h

```diff
@@ -33,7 +33,7 @@ constexpr std::array<int, 2> VERSION_PARTICLE_RESTART {2, 0};
 constexpr std::array<int, 2> VERSION_TRACK {2, 0};
 constexpr std::array<int, 2> VERSION_SUMMARY {6, 0};
 constexpr std::array<int, 2> VERSION_VOLUME {1, 0};
-constexpr std::array<int, 2> VERSION_VOXEL {1, 0};
+constexpr std::array<int, 2> VERSION_VOXEL {1, 1};
```

Comment:
I would actually bump the major version. Bumping minor would be reserved for, say, additions to a file (as opposed to fundamentally changing the meaning of a dataset).

Reply:
Good call. Can do!